specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -553,6 +950,14 @@
       "x-ms-parameter-grouping": { "name": "search-request-options" },
       "x-ms-parameter-location": "method"
     },
+    "ClientAcceptParameter": {
```

Comment:
Do we need to set this explicitly? I thought AutoRest handled this for us?

Reply:
It was because I needed to set odata.metadata = none (See DocumentsOperations.cs) and found no other way to do that. This was set in custom code. 

---
In reply to: [208103195](https://github.com/Azure/azure-rest-api-specs/pull/3578#discussion_r208103195) [](ancestors = 208103195)